src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyenAccount.js

```diff
@@ -1,7 +1,9 @@
 const { onFieldValid, onBrand } = require('./commons/index');
 const store = require('../../../store');
 
-const cardNode = document.getElementById('card');
+let checkout;
```

Comment:
Can't we assign the `checkout` variable already here?

Reply:
the declaration of the checkout variable depends on `store.checkoutConfiguration`, which implements `handleAction()` which calls `checkout`. 
If I use checkout before I declare it linting will throw errors.